config/sync/user.role.os2loop_user_document_author.yml

```diff
@@ -16,6 +16,8 @@ permissions:
   - 'create os2loop_documents_document content'
   - 'download any image files'
   - 'edit own os2loop_documents_document content'
+  - 'flag os2loop_upvote_correct_answer'
+  - 'unflag os2loop_upvote_correct_answer'
```

Comment:
Does this make sense? We don't have answers/comments om documents, so I'm not sure this permission makes sense for a document author.